File: API/FunctionAppDoc2Data/Models/ReceiptVerificationMasterDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FunctionAppDoc2Data.Models;
public class ReceiptVerificationMasterDTO
{
    public Guid ReceiptId { get; set; }
    public Guid UserId { get; set; }
    public string VendorName { get; set; }
    public string VendorAddress { get; set; }
    public string VendorPhone { get; set; }
    public string VendorEmail { get; set; }
    public string CustomerName { get; set; }
    public string CustomerAddress { get; set; }
    public string CustomerPhone { get; set; }
    public string InvoiceNumber { get; set; }
    public DateTime InvoiceDate { get; set; }
    public decimal SubTotal { get; set; }
    public decimal TaxAmount { get; set; }
    public decimal Total { get; set; }
    public string ImagePath { get; set; }
    public string Image { get; set; }
    public string OriginalFileName { get; set; }
    public bool IsImage { get; set; }
    public List<ReceiptVerificationItemsDTO> ReceiptVerificationItems { get; set; }
}

File: src/app/interfaces/receipt-verification-master-dto.ts
import { ReceiptVerificationItemsDTO } from "./receipt-verification-items-dto";

export interface ReceiptVerificationMasterDTO {
    receiptId: string;
    userId: string;
    vendorName: string;
    vendorAddress: string;
    vendorPhone: string;
    vendorEmail: string;
    customerName: string;
    customerAddress: string;
    customerPhone: string;
    invoiceNumber: string;
    invoiceDate: string;
    subTotal: number;
    taxAmount: number;
    total: number;
    imagePath: string;
    image: string;
    originalFileName: string;
    isImage: boolean;
    receiptVerificationItems: ReceiptVerificationItemsDTO[] ;
}
